cmd/run.go

```diff
@@ -124,7 +124,7 @@ func init() {
 	runCmd.Flags().IntVarP(&flgs.ShardN, "shard-n", "", 0, flgs.Usage("ShardN"))
 	runCmd.Flags().IntVarP(&flgs.Random, "random", "", 0, flgs.Usage("Random"))
 	runCmd.Flags().StringVarP(&flgs.Format, "format", "", "", flgs.Usage("Format"))
-	runCmd.Flags().BoolVarP(&flgs.Profile, "profile", "", false, flgs.Usage("Profile"))
+	runCmd.Flags().BoolVarP(&flgs.DisableProfile, "disable-profile", "", false, flgs.Usage("DisableProfile"))
```

Comment:
```suggestion
	runCmd.Flags().BoolVarP(&flgs.DisableProfile, "disable-profile", "", true, flgs.Usage("DisableProfile"))
```
What is the opposite of the opposite? Difficult.. 😅

Reply:
> Profile is default enabled

I was wrong.